cloudside/tests/test_viz.py

```diff
@@ -14,7 +14,7 @@
 
 
 IMG_OPTS = dict(
-    tolerance=int(os.environ.get("MPL_IMGCOMP_TOLERANCE", 25)),
+    tolerance=int(os.environ.get("MPL_IMGCOMP_TOLERANCE", 37)),
```

Comment:
I've pretty come around to the idea that image testing is a fool's errand for anyone but put viz libraries.

Reply:
perhaps you're right, but let's drop these in a patch release. For today I'm just tidying up what's already here. I also think the --mpl image check is really cool and helps us find unexpected bugs.